packages/core/src/modules/company/company-event-service.ts

```diff
@@ -0,0 +1,26 @@
+import { Company, Event } from "@dotkomonline/types"
+import { Cursor } from "../../utils/db-utils"
+import { CompanyEventRepository } from "./company-event-repository" // Note: you might need to create or rename this file based on previous changes.
+
+export interface CompanyEventService {
+  createEvent: (company: Company["id"], event: Event["id"]) => Promise<void>
+  deleteEvent: (company: Company["id"], event: Event["id"]) => Promise<void>
+  getEventsByCompanyId: (company: Company["id"], take: number, cursor?: Cursor) => Promise<Event[]>
```

Comment:
To stay consistent lets use interface method syntax

```ts
interface Foo {
  method(args: T): R
}

Reply:
Agreed!